oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elasticsearch/query/ElasticsearchResultRowIterator.java

```diff
@@ -222,27 +225,28 @@ private ElasticsearchSearcher getCurrentSearcher(ElasticsearchIndexNode indexNod
         return new ElasticsearchSearcher(indexNode);
     }
 
-    private FulltextIndex.FulltextResultRow convertToRow(SearchHit hit) throws IOException {
+    private FulltextIndex.FulltextResultRow convertToRow(SearchHit hit,
+                                                         ElasticsearchFacetProvider elasticsearchFacetProvider) throws IOException {
         String id = hit.getId();
         String path = idToPath(id);
         if (path != null) {
             if ("".equals(path)) {
                 path = "/";
             }
-            if (pr.isPathTransformed()) {
+            if (this.pr.isPathTransformed()) {
```

Comment:
Not quite sure why "this." 
Related, a field named "pr" is strange... I think it should be renamed?